pkg/controllers/sync/controller.go

```diff
@@ -318,7 +321,7 @@ func (s *SyncController) HasSynced() bool {
 	}
 
 	if s.typeConfig.GetRevisionHistoryEnabled() && !s.revListerSynced() {
-		klog.V(2).Infof("controllerRevision list not synced")
+		s.logger.V(3).Info("controllerRevision list not synced")
```

Comment:
MUST begin with a capital letter (with the exception of identifiers that conventionally begin with a lowercase).

Reply:
done, pls take review.